torchaudio/functional/functional.py

```diff
@@ -479,6 +480,127 @@ def _mel_to_hz(mels: Tensor, mel_scale: str = "htk") -> Tensor:
     return freqs
 
 
+def _hz_to_bark(freq: float, bark_scale: str = "traunmuller") -> float:
```

Comment:
I think we can have `bark_scale` be required variables in `_hz_to_bark` and `_bark_to_hz`, as these are internal functions called only within the barkscale function. this way, we ensure that we do pass in the correct value every time.

Reply:
To clarify, the functions `_hz_to_bark` and `_bark_to_hz` are called within the function `barkscale_fbanks` and not within `bark_scale`.

Regarding your request, `_hz_to_bark` and `_bark_to_hz` can be defined using only required variables, but that would not keep the coherence with the functions `_hz_to_mel` and `_mel_to_hz`. Is it recommendable?